infra/api/service/main.tf

```diff
@@ -138,7 +138,7 @@ module "service" {
     }
   } : null
 
-  extra_environment_variables = local.service_config.extra_environment_variables
+  extra_environment_variables = merge(local.service_config.extra_environment_variables, { "ENVIRONMENT_NAME" : var.environment_name })
```

Comment:
Could we call the variable `ENVIRONMENT`? That's what we're expecting in the API code

Reply:
@chouinar done!